configure.ac

```diff
@@ -1638,6 +1644,7 @@ if test "$enable_threads" = "multiple" ; then
     esac
 fi
 AC_DEFINE_UNQUOTED([MPICH_THREAD_GRANULARITY],$thread_granularity,[Method used to implement atomic updates and access])
+AC_DEFINE_UNQUOTED([MPICH_THREAD_USE_MDTA],$thread_mdta,[Optimizing critical section with message-driven thread activation])
```

Comment:
I'd rather this just be defined or not defined. Adding a value is useful for macros with different levels, but in this case we are just dealing with a boolean.